tests/test_api.py

```diff
@@ -1,119 +1,62 @@
-    assert netcdf_bbox(filepath) == [0.0, -90.0, 357.5, 90.0]
+    assert geoextent.getMetadata(filepath, 'bbox' , True) == [[-90.0, 0.0, 90.0, 357.5],['2002-07-31']]
+
+
+def test_kml_extract_bbox():
+    filepath=__location__+"/Testdata/aasee.kml"
+    assert geoextent.getMetadata(filepath, 'bbox' , True) == [[7.594213, 51.942466, 7.618246, 51.957278],[None]]
+
+def test_geotiff_extract_bbox():
+    filepath=__location__+"/Testdata/wf_100m_klas.tif"
+    assert geoextent.getMetadata(filepath, 'bbox' , True) == [[5.9153007564753155, 50.31025197410836, 9.468398712484145, 52.5307755328733],[None]]
+
+def test_gpkg_extract_bbox():
+    filepath=__location__+"/Testdata/census2016_cca_qld_short.gpkg"
+    assert geoextent.getMetadata(filepath, 'bbox' , True) == [[-43.7405, 96.8169, -9.14218, 167.998],[None]]
+
+def test_csv_extract_bbox():
+    filepath=__location__+"/Testdata/folder/cities_NL.csv"
+    assert geoextent.getMetadata(filepath, 'bbox' , True) == [[6.574722, 51.434444, 4.3175, 53.217222],['2018-09-30']]]
+
+def test_gml_extract_bbox():
+    filepath=__location__+"/Testdata/clc_1000_PT.gml"
+    assert geoextent.getMetadata(filepath, 'bbox' , True) == [[-17.5420724159224, 32.3966928193202, -6.95938792923511, 39.3011352746141],['2013-11-30T23:00:00Z']]
 
 def test_shapefile_extract_bbox():
     filepath=__location__+"/Testdata/Abgrabungen_Kreis_Kleve_Shape.shp"
-    sf = shapefile.Reader(filepath)
-    output = sf.bbox
-    print(output)
-    assert output == [295896.274870878, 5694747.64703736, 325999.79578122497, 5747140.98659967]
-
-###############################
-# Functions
-###############################
-"""
-Function for extracting the bbox of netcdf file
-
-:param filepath: path to the file
-:returns: bounding box of the netCDF in the format [minlon, minlat, maxlon, maxlat]
-"""
-def netcdf_bbox(filepath):
-    ds = xr.open_dataset(filepath)
-    try:
-        lats = ds.coords["lat"]
-        lons = ds.coords["lon"]
-    except Exception as e:
-        try:
-            lats = ds.coords["latitude"]
-            lons = ds.coords["longitude"]
-        except Exception as e:
-            click.echo(e)
-
-    min_lat=min(lats).values
-    min_lat_float=float(min_lat)
-    min_lon=min(lons).values
-    min_lon_float=float(min_lon)
-    max_lat=max(lats).values
-    max_lat_float=float(max_lat)
-    max_lon=max(lons).values
-    max_lon_float=float(max_lon)
-
-    bbox = [min_lon_float,min_lat_float,max_lon_float,max_lat_float]
-    ds.close()
-    return bbox
\ No newline at end of file
+    assert geoextent.getMetadata(filepath, 'bbox' , True) == [[295896.274870878, 5694747.64703736, 325999.79578122497, 5747140.98659967],[None]]
+
+def test_json_extract_bbox():
+    filepath=__location__+"/Testdata/folder/schutzhuetten_aachen.json"
+    assert geoextent.getMetadata(filepath, 'bbox' , True) == [[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454],[None]]
```

Comment:
Please also add a GeoJSON file with timestamps as `properties`, see

- https://gis.stackexchange.com/questions/162081/representing-time-in-geojson
- GDAL driver notes on recognized fields: https://gdal.org/drivers/vector/geojson.html

It would be great if the extraction supports the same list of field names as for CSV here, e.g. `time, date, timestamp, datetime, ...` (all case insensitive).